### ISSUE TITLE
review alternative links 2021/03/20

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#11369]
@garfield69 @ilike2burnthing

- [ ] 1337x || https://1337x.unblocked.monster/ || Bad HTTP response code: 301
- [x] anidex || https://anidex.info/ || Bad HTTP response code: 403
- [x] borgzelle || https://borgzelle.borg/ || HTTPSConnectionPool(host='borgzelle.borg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Name or service not known'))
- [x] btdigg || https://archive.today/ua/ || Bad HTTP response code: 429
- [x] cpasbien || https://wwwv.cpasbien-fr.fr/ || Bad HTTP response code: 403
- [x] dicmusic || https://dicmusic.club/ || Bad HTTP response code: 301
- [x] dxdhd || https://dxdhd.com/ || HTTPSConnectionPool(host='dxdhd.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
- [x] ebooks-shares || https://ebooks-shares.org/ || Bad HTTP response code: 504
- [x] empornium || https://www.empornium.me/ || HTTPSConnectionPool(host='www.empornium.me', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(, 'Connection to www.empornium.me timed out. (connect timeout=60)'))
- [x] empornium2fa || https://www.empornium.me/ || HTTPSConnectionPool(host='www.empornium.me', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(, 'Connection to www.empornium.me timed out. (connect timeout=60)'))
- [x] epizod || https://wwv.epizod.tv/ || Bad HTTP response code: 403
- [ ] ettv || https://ettv.unblocked.monster/ || Bad HTTP response code: 301
- [x] extratorrent-it || https://extratorrent.unblockit.buzz/ || Bad HTTP response code: 302
- [x] extratorrent-it || https://extratorrent2.unblockninja.com/ || Bad HTTP response code: 301
- [ ] eztv || https://eztv.unblocked.monster/ || Bad HTTP response code: 301
- [x] firebit || https://firebit.club/ || HTTPSConnectionPool(host='firebit.club', port=443): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Name or service not known'))
- [x] funreleases || https://funreleases.me/ || Bad HTTP response code: 521
- [ ] glodls || https://glodls.unblocked.monster/ || Bad HTTP response code: 301
- [x] hdonly || https://hd-only.org/ || Bad HTTP response code: 500
- [x] hdchina || https://hdchina.org/ || Bad HTTP response code: 302
- [x] hdhome || https://hdhome.org/ || Bad HTTP response code: 302
- [x] hdtorrentsit || https://hdtorrents.xyz/ || Bad HTTP response code: 404
- [x] hdzone || https://hdzone.me/ || Bad HTTP response code: 302
- [x] hebits || https://hebits.net/ || HTTPSConnectionPool(host='hebits.net', port=443): Read timed out. (read timeout=60)
- [ ] idope || https://idope.unblocked.monster/ || Bad HTTP response code: 301
- [x] kickasstorrents-ws || https://kick4ss.com/ || Bad HTTP response code: 302
- [x] mteamtp || https://pt.m-team.cc/ || Bad HTTP response code: 302
- [ ] magnet4you || https://magnet4you.unblocked.monster/ || Bad HTTP response code: 301
- [x] magnetico || http://127.0.0.1:8080/ || HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 111] Connection refused'))
- [x] mteamtp2fa || https://pt.m-team.cc/ || Bad HTTP response code: 302
- [x] notwhatcd || https://notwhat.cd/ || Bad HTTP response code: 522
- [x] oshenpt || https://www.oshen.win/ || Bad HTTP response code: 302
- [ ] oxtorrent || https://oxtorrent.unblocked.monster/ || Bad HTTP response code: 301
- [ ] rutor || https://rutor.unblocked.monster/ || Bad HTTP response code: 301
- [x] shellife || https://shellife.eu/ || HTTPSConnectionPool(host='shellife.eu', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(, 'Connection to shellife.eu timed out. (connect timeout=60)'))
- [x] thepiratebay || https://tpb19.ukpass.co/ || Bad HTTP response code: 301
- [x] thepiratebay || https://pirateproxy.tube/ || Bad HTTP response code: 301
- [x] tjangto || https://www.jjangtorrent.com/ || Bad HTTP response code: 301
- [ ] tokyotosho || https://tokyotosho.unblocked.monster/ || Bad HTTP response code: 301
- [x] torlock || https://torlock.unblockit.link/ || Bad HTTP response code: 301
- [ ] torrentdownloads || https://torrentdownloads.unblocked.monster/ || Bad HTTP response code: 301
- [ ] torrent9 || https://www.oxtorrent.me/ || Bad HTTP response code: 522
- [ ] torrent9clone || https://torrent9.unblocked.monster/ || Bad HTTP response code: 301
- [x] torrentccf || https://et8.org/ || Bad HTTP response code: 302
- [ ] torrentdownload || https://torrentdownload.unblocked.monster/ || Bad HTTP response code: 301
- [ ] torrentgalaxy || https://tgx.unblocked.monster/ || Bad HTTP response code: 301
- [x] torrentqq || https://torrentqq78.com/ || Bad HTTP response code: 301
- [x] torrentz2 || https://torrentz.unblockit.buzz/ || Bad HTTP response code: 301
- [x] yingk || https://yingk.com/ || HTTPSConnectionPool(host='yingk.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -5] No address associated with hostname'))
- [ ] zooqle || https://zooqle.unblocked.monster/ || Bad HTTP response code: 301

**Comment by ilike2burnthing** [comment]
**Working - no action needed, but FYI**
anidex - regularly fails, ongoing site issue
cpasbien - region locked
dicmusic - login page closed, redirects to https://kshare.club/ - https://github.com/Jackett/Jackett/pull/9041#issuecomment-748673099
epizod - has had issues previously FlareSolverr/FlareSolverr#25
https://extratorrent.unblockit.buzz/ - redirects to /home
magnetico - local DHT
https://extratorrent2.unblockninja.com/ - removed https://github.com/Jackett/Jackett/commit/dffe9fe0cf2d2606550367bd6b29619d7eea005e
https://firebit.club/ - removed https://github.com/Jackett/Jackett/commit/b1a4fc4fa6299187cceeffbd4a6e3bd2a621f3e2
https://www.jjangtorrent.com/ - replaced with https://www.jjangt.com/ https://github.com/Jackett/Jackett/commit/64d3fbaec4cfe4570c9921879298e7f6f7c2152f
https://tpb19.ukpass.co/ - replaced with tpb22.* https://github.com/Jackett/Jackett/commit/b54718b838e77084a210e6b263558b541ab8dee6
https://pirateproxy.tube/ - replaced with *.ltda https://github.com/Jackett/Jackett/commit/b54718b838e77084a210e6b263558b541ab8dee6
https://torlock.unblockit.link/ - replaced with *.buzz https://github.com/Jackett/Jackett/commit/1ef93f9bee83bb30f99ef41c70d6583507815233
https://torrentqq78.com/ - replaced with *79.com https://github.com/Jackett/Jackett/commit/cd73fe2507bdab0600abbecf17052ba8c564ba98

**Site is up, but I don't have an account to test**
borgzelle - no public DNS, IP must be set in hosts #9855
ebooks-shares
hdonly
hdchina
hdhome
hdtorrentsit - region locked
hebits
mteamtp
torrentccf
_[assuming a lot of these are causing errors because of redirect to login page]_

**Covered by other open issues**
btdigg - works for me when using a VPN #10579 though we don't use _archive.today_
dxdhd - down https://github.com/Jackett/Jackett/issues/4035
empornium - down https://github.com/Jackett/Jackett/issues/11372
funreleases - https://github.com/Jackett/Jackett/issues/9812
notwhatcd - https://github.com/Jackett/Jackett/issues/818
shellife - https://github.com/Jackett/Jackett/issues/721
yingk - https://github.com/Jackett/Jackett/issues/8823

**To Do**
https://unblocked.monster/ is currently down - check later
https://www.oxtorrent.me/ - timing out, check later

**Comment by ngosang** [comment]
@ilike2burnthing thank you! The tool I'm running to generate this list is very basic and several links are false positives. You can close the issue when you think it's fine.

**Comment by ilike2burnthing** [comment]
I've opened a new issue for unblocked.monster and added oxtorrent.me to the already open issue for torrent9.to so I'll close this one now
